public/components/layout.js

```diff
@@ -305,9 +317,10 @@ class GeolocatorPreview extends HTMLElement {
                 wrapper = {
                     "@context": ["http://www.w3.org/ns/anno.jsonld", "https://geojson.org/geojson-ld/geojson-context.jsonld"],
                     "type": "Annotation",
+                    "creator": userObj["creator"],
```

Comment:
Make sure not to add the `creato`r key if the value is an empty string, undefined, or null